styles/Home.module.css

```diff
@@ -62,6 +66,12 @@
   }
 }
 
+hr.sectionSeparator {
+  width: calc(100% - 40px);
+  margin: 6rem 0;
+  border-color: #000;
+}
+
```

Comment:
This could be done via styled components so it can be reused:
```
const SectionSeparator = styled.hr`
  ...
`
```

Reply:
I made a new component to make it reusable